packages/cmf/src/sagas/http.js

```diff
@@ -47,7 +47,14 @@ export class HTTPError extends Error {
  * @param  {Response} response A response object
  * @return {Promise}           A promise that resolves with the result of parsing the body
  */
-export function handleBody(response) {
+export function handleBody(response, method) {
```

Comment:
I would prefer to have method as { method } as in other below. So it look like handleBody(response, request) {}
WDYT  ?

Reply:
yep ok, thanks